Makefile

```diff
@@ -51,6 +51,9 @@ all:
 	KUBEEDGE_OUTPUT_SUBPATH=$(OUT_DIR) hack/make-rules/build.sh $(WHAT)
 endif
 
+.PHONY: compile
```

Comment:
It should be part of make with `export BUILD_WITH_CONTAINER=true`
make all or make is also do the compilation work